lib/types/src/lib.rs

```diff
@@ -105,7 +105,7 @@ pub use value::{RawValue, ValueType};
 pub use crate::libcalls::LibCall;
 pub use crate::memory::MemoryStyle;
 pub use crate::table::TableStyle;
-pub use crate::trapcode::TrapCode;
+pub use crate::trapcode::{OnCalledAction, TrapCode};
```

Comment:
Can we add a new comment linking to the asyncify refactor afterwards:

```
// TODO: issue #XYZ 
```

Reply:
done